packages/devextreme/js/__internal/ui/m_drop_down_button.ts

```diff
@@ -412,6 +414,11 @@ class DropDownButton extends Widget<DropDownButtonProperties> {
         at: `${horizontalAlignment} bottom`,
       },
       _wrapperClassExternal: DROP_DOWN_EDITOR_OVERLAY_CLASS,
+      integrationOptions: {
+        templates: {
+          content: undefined,
```

Comment:
If the reason is "default template name is the same for component and overlay", isn't then this issue actual for another components? Particularly, dropDownBox